spec/controllers/api/v1/bands_controller_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+describe Api::V1::BandsController, type: :controller do
+    let!(:band) { Band.create(name: "test_band", biography: "test_band_time") }
+
+    describe "GET#index" do
+        it "should return all bands" do
+
+            get :index
+            returned_json = JSON.parse(response.body)
+    
+            expect(response.status).to eq 200
+            expect(response.content_type).to eq("application/json; charset=utf-8")
+            
+            binding.pry
+            
+            expect(returned_json.length).to eq(1)
+            returned_json.first["name"] = band.name
```

Comment:
Not sure what the purpose of this line is that comes after the `expect` statement